doppler/models.go

```diff
@@ -56,3 +56,14 @@ func parseSecretId(id string) (project string, config string, name string, err e
 	}
 	return tokens[0], tokens[1], tokens[2], nil
 }
+
+type Project struct {
+	Id          string `json:"id"`
```

Comment:
Sure thing! To confirm, we want to change our variable names to `Slug` even though we're parsing `json:"id"`? I can definitely make that happen

Reply:
Good point. It's unfortunate that updating our API is a breaking change b/c otherwise we would have switched to `slug`. Maybe the right approach would be to add a new `slug` field to our APIs that has the same value as `id`. In either case, I'd prefer we only expose `slug` for all new things